Libraries/Microsoft.Teams.Apps/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.Teams.Apps.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100fda1d1b3a8f1e375020aeedb26e512a71e0cdd9c854e4c87e181c1152967e79b5ab815e18e0af0f036e06d3dc0811fdb7251bce2fbec8b9492d7e8bc6a68e6bd3ca28c4c4a30b65c5915268b56644cd4b8e870a63baa7709f1d5a445c8a381d7b3ab3f116410081ec9f6dde59785edf01b689d14aeda42ae4e33139bf1ff92b0")]
```

Comment:
Since Microsoft.Teams.Apps is strong name signed, exposing its internals is only allowed to strong name signed packages. In this case the test project is strong name signed w/ the same public key. 

Think of the public key as an extension of the package name. Nothing to do with authentication.